src/platform/nrfconnect/OTAImageProcessorImpl.cpp

```diff
@@ -184,7 +181,11 @@ CHIP_ERROR OTAImageProcessorImpl::Apply()
             [](System::Layer *, void * /* context */) {
                 PlatformMgr().HandleServerShuttingDown();
                 k_msleep(CHIP_DEVICE_CONFIG_SERVER_SHUTDOWN_ACTIONS_SLEEP_MS);
+#ifdef CONFIG_DFU_TARGET_SUIT
+                dfu_target_suit_reboot();
```

Comment:
The purpose of `Reboot` was also to save the reboot reason to "software update". Is this somehow handled for SUIT?

Reply:
No, it is not, we need to think how to handle it. I'm going to do it in the next iteration after the release. For now, it is base support.